react/src/components/organisms/RichText/RichText.stories.js

```diff
@@ -1,76 +1,39 @@
+import { action } from '@storybook/addon-actions';
 
-import {
-  Paragraph,
-  Heading,
-  CompHeading,
-  SidebarHeading,
-  DecorativeLink,
-  UnorderedList,
-  compHeadingOptions,
-  sidebarHeadingOptions,
-  decorativeLinkOptions,
-  unorderedOptions
-} from './RichText.require';
 import RichText from './index';
 import RichTextDocs from './RichText.md';
 
 storiesOf('organisms', module)
   .addDecorator(withKnobs({ escapeHTML: false }))
   .add(
     'RichText', () => {
-      const decoreOptionsWithKnobs = Object.assign(...Object.entries(decorativeLinkOptions).map(([k, v]) => (
-        { [k]: v(DecorativeLink.defaultProps[k]) })));
-      const compOptionsWithKnobs = Object.assign(...Object.entries(compHeadingOptions).map(([k, v]) => (
-        { [k]: v(CompHeading.defaultProps[k]) })));
-      const sideOptionsWithKnobs = Object.assign(...Object.entries(sidebarHeadingOptions).map(([k, v]) => (
-        { [k]: v(SidebarHeading.defaultProps[k]) })));
-      const paraOptionsWithKnobs = {
-        text: text('Paragraph1: text', 'A <strong>paragraph</strong> (from the Greek paragraphos, "to write beside" or "written beside") is a self-contained unit of a discourse in writing dealing with a particular point or idea. A paragraph consists of one or more sentences. Though not required by the syntax of any language, paragraphs are usually an expected part of formal writing, used to organize longer prose.', 'Paragraph')
-      };
-      const orderedOptionsWithKnobs = Object.assign(...Object.entries(unorderedOptions).map(([k, v]) => (
-        { [k]: v(UnorderedList.defaultProps[k]) })));
-      const headerIndent = boolean('headerIndent', RichText.defaultProps.headerIndent, 'RichText');
-      const anchorLinks = boolean('anchorLinks', RichText.defaultProps.anchorLinks, 'RichText');
-      const description = {
-        rteElements: [{
-          path: '@atoms/11-text/paragraph.twig',
-          data: {
-            paragraph: {
-              text: text('rteParagraph', 'Optional description', 'RichText')
-            }
-          }
-        },
-        {
-          path: '@atoms/08-lists/unordered-list.twig',
-          data: {
-            unorderedList: [
-              { text: 'This is a list item in an unordered list' },
-              { text: 'An unordered list is a list in which the sequence of items is not important. Sometimes, an unordered list is a bulleted list. And this is a long list item in an unordered list that can wrap onto a new line.' },
-              {
-                text: 'Lists can be nested inside of each other',
-                sublist: [
-                  { text: 'This is a nested list item' },
-                  { text: 'This is another nested list item in an unordered list' }
-                ]
-              },
-              { text: 'This is the last list item' }
-            ]
-          }
-        }]
+      const props = {
+        rawHtml: text('rawHtml', '<h1>This is heading 1</h1>'),
```

Comment:
you include this `knob` but it is never used or has no visible change in the preview. Maybe I am misunderstanding it's use?

Reply:
This is because if children are passed, it will render children nodes rather than the rawHtml string. I added a conditional check there so it's less confusing in storybook